homeassistant/components/freebox/camera.py

```diff
@@ -0,0 +1,217 @@
+        if (node["category"] != "camera") or (nodeid in tracked):
+            continue
+        new_tracked.append(FreeboxCamera(hass, router, node))
+        tracked.add(nodeid)
+
+    if new_tracked:
+        async_add_entities(new_tracked, True)
+
+
+class FreeboxCamera(FreeboxHomeBaseClass, FFmpegCamera):
+    """Representation of a Freebox camera."""
+
+    _attr_should_poll = True
+
+    def __init__(
+        self, hass: HomeAssistant, router: FreeboxRouter, node: dict[str, Any]
+    ) -> None:
+        """Initialize a camera."""
+
+        super().__init__(hass, router, node)
+        self._flip: Any
+        device_info = {
+            CONF_NAME: node["label"].strip(),
+            CONF_INPUT: node["props"]["Stream"],
+            CONF_EXTRA_ARGUMENTS: DEFAULT_ARGUMENTS,
+        }
+        FFmpegCamera.__init__(self, hass, device_info)
+
+        self._supported_features = (
+            CameraEntityFeature.ON_OFF | CameraEntityFeature.STREAM
+        )
+
+        self._command_motion_detection = self.get_command_id(
+            node["type"]["endpoints"], ATTR_DETECTION
+        )
+        self._high_quality_video = self.get_command_id(
+            node["show_endpoints"], ATTR_QUALITY
+        )
+        self._command_flip = self.get_command_id(node["show_endpoints"], ATTR_FLIP)
+        self._motion_threshold = self.get_command_id(
+            node["show_endpoints"], ATTR_THRESHOLD
+        )
+        self._motion_sensitivity = self.get_command_id(
+            node["show_endpoints"], ATTR_SENSITIVITY
+        )
+        self._activation_with_alarm = self.get_command_id(
+            node["show_endpoints"], ATTR_ACTIVATION
+        )
+        self._timestamp = self.get_command_id(node["show_endpoints"], ATTR_TIMESTAMP)
+        self._volume_micro = self.get_command_id(node["show_endpoints"], ATTR_VOLUME)
+        self._sound_detection = self.get_command_id(
+            node["show_endpoints"], ATTR_SOUND_DETECTION
+        )
+        self._sound_trigger = self.get_command_id(
+            node["show_endpoints"], ATTR_SOUND_TRIGGER
+        )
+        self._rstp = self.get_command_id(node["show_endpoints"], ATTR_RTSP)
+        self._disk = self.get_command_id(node["show_endpoints"], ATTR_DISK)
+
+        self.update_node(node)
+
+    @property
+    def supported_features(self) -> CameraEntityFeature:
+        """Flag supported features."""
+        return self._supported_features
+
+    @property
+    def flip(self) -> bool:
+        """Return flip."""
+        return self._flip
+
+    @property
+    def command_flip(self) -> Any | str:
+        """Return command_flip."""
+        return self._command_flip
+
+    def set_flip(self, state: bool) -> None:
+        """Update flip."""
+        self._flip = state
+
+    async def async_flip(self, entity: FreeboxCamera) -> None:
+        """Flip the camera stream."""
+        entity.set_flip(not entity.flip)
+        await entity.set_home_endpoint_value(entity.command_flip, entity.flip)
```

Comment:
```suggestion
    async def async_flip(self, entity: FreeboxCamera) -> None:
        """Flip the camera stream."""
        self._flip = not entity.flip
        await entity.set_home_endpoint_value(entity._command_flip, entity.flip)
```

I don't think you need to overload the entity with command_flip property and set_flip fonction.